pre/remap_restart/remap_catchANDcn.py

```diff
@@ -225,6 +244,195 @@ def copy_merra2(self):
     print("Copy file "+f +" to " + rst_dir)
     shutil.copy(f, dest)
 
-if __name__ == '__main__' :
-   catch = catchANDcn(params_file='remap_params.yaml')
+def ask_catch_questions():
+   catch_input_shared_rst_dir = ''
+   def has_rs_rc_out(path):
+     if os.path.exists(path):
+        dirs = os.listdir(path)   
+        if 'rs' in dirs and 'rc_out' in dirs:
+           nonlocal catch_input_shared_rst_dir
+           catch_input_shared_rst_dir = path
+           return True
+     return False
+
+   def has_catch_rst(text):
+     if len(text) !=10: return False
+     yyyy = text[0:4]
+     mm   = text[4:6]
+     dd   = text[6:8]
+     hh   = text[8:10]
+     nonlocal catch_input_shared_rst_dir
+     rst_dir   = catch_input_shared_rst_dir + '/rs/ens0000/Y'+yyyy +'/M'+mm+'/'
```

Comment:
I think the hardwired path here is specific to GEOSldas, and it only allows for remapping of ens member #0000.  I added "GEOSldas" into the "usage" text for "land_only" in the hope of making this limitation a bit clearer to the user.